docs/development/releasing.rst

```diff
@@ -157,19 +138,14 @@ packages that use the full bugfix/maintenance branch approach.)
       Also, it might be tempting to use the ``--tags`` argument to ``git push``,
       but this should *not* be done, as it might push up some unintended tags.
 
-#. Build and test the Astropy wheels.  See the `wheel builder README
-   <https://github.com/MacPython/astropy-wheels>`_ for instructions.  In
-   summary, clone the wheel-building repo, edit the ``.travis.yml``
-   text file with the branch or commit for the release,
-   commit and then push back up to github.  This will trigger a wheel build
-   and test on OSX, Linux, and Windows. Check the build has passed on on the
-   Travis-CI interface at https://travis-ci.org/MacPython/astropy-wheels.
-   You'll need commit privileges to the ``astropy-wheels`` repo; ask Tom Kooij
-   or on the mailing list if you do not have them.
+  At this point if all goes well, the wheels and sdist will be build
+  in the `Azure core package pipeline`_ and uploaded to PyPI!
 
-#. If the tests do *not* pass, you'll have to fix whatever the problem is.
-   First you will need to back out the release procedure by dropping the commits
-   you made for release and removing the tag you created::
+#. In the unlikely event there are any issues with the wheel building for the tag
```

Comment:
```suggestion
#. In the event there are any issues with the wheel building for the tag
```
This may look like trolling, but it's in the docs style guide to try to avoid "extraneous"/judgemental  language (https://docs.astropy.org/en/stable/development/style-guide.html#a-note-about-style-and-tone) - this is debatable, but I feel it applies here.